xml/depl_conf_admin_crowbar.xml

```diff
@@ -30,12 +30,12 @@
   <title><guimenu>User Settings</guimenu></title>
 
   <para>
-   In this section, you can manage users for the &crow; Web interface. The user
-   <systemitem class="username">crowbar</systemitem> (password
-   <literal>crowbar</literal>) is preconfigured. Use the
-   <guimenu>Add</guimenu>, <guimenu>Edit</guimenu>, and
-   <guimenu>Delete</guimenu> buttons to manage user accounts. Users configured
-   here have no relation to existing system users on the &admserv;.
+   In this section, you can manage the administration user for the &crow; Web
+   interface. Use the entries to change the user name and the password. The
```

Comment:
web*

Reply:
Actually, the doc always uses Web, so I guess I should stay consistent here.